README.md

```diff
@@ -312,6 +312,8 @@ Grist can be configured in many ways. Here are the main environment variables it
 | GRIST_UI_FEATURES                  | comma-separated list of UI features to enable. Allowed names of parts: `helpCenter,billing,templates,createSite,multiSite,multiAccounts,sendToDrive,tutorials,supportGrist`. If a part also exists in GRIST_HIDE_UI_ELEMENTS, it won't be enabled.                                                                                                            |
 | GRIST_UNTRUSTED_PORT               | if set, plugins will be served from the given port. This is an alternative to setting APP_UNTRUSTED_URL.                                                                                                                                                                                                                                                      |
 | GRIST_WIDGET_LIST_URL              | a url pointing to a widget manifest, by default `https://github.com/gristlabs/grist-widget/releases/download/latest/manifest.json` is used                                                                                                                                                                                                                    |
+| GRIST_LOG_SKIP_HTTP                | When set to `false`, log HTTP requests and responses information. Defaults to `true`.                                                                                                                                                                                                                                                                         |
```

Comment:
Can you remind me why the double negative? Is there a simpler to read pair of variables that could be used instead?

Reply:
So Defaults to `true` sounds like `GRIST_LOG_SKIP_HTTP` is on by default, so normally, without setting anything, http logs are skipped, so not emitted, is that correct?

Reply:
> Can you remind me why the double negative? Is there a simpler to read pair of variables that could be used instead?

That's an existing environment variable that I document.
I agree that the naming is not ideal, I would be glad to rename it or deprecate and make another variable.

> So Defaults to true sounds like GRIST_LOG_SKIP_HTTP is on by default, so normally, without setting anything, http logs are skipped, so not emitted, is that correct?

You're right!